ocaml/typing/includecore.ml

```diff
@@ -492,16 +492,11 @@ let report_type_mismatch first second decl env ppf err =
 
 let compare_global_flags flag0 flag1 =
   match flag0, flag1 with
-  | Global, (Nonlocal | Unrestricted) ->
+  | Global, Unrestricted ->
     Some {order = First; nonlocal = false}
```

Comment:
The `nonlocal` field should be removed. (Should also generally grep the source for "nonlocal" to see if there are any other leftovers.)